client/src/main/java/com/tencent/rss/client/impl/ShuffleWriteClientImpl.java

```
@@ -311,6 +313,9 @@ public Roaring64NavigableMap getShuffleResult(String clientType, Set<ShuffleServ
           blockIdBitmap = response.getBlockIdBitmap();
           isSuccessful = true;
           break;
+        } else if (response.getStatusCode() == ResponseStatusCode.INTERRUPTED) {
```

Our aim is to change the status of tasks from failed to success?

---

The target of PR is to ignore InterrptException, not mark is as successful

---

Ignore the exception will make the tasks that should be killed, will be successful.

---

InterruptionException may be caused by another reason, not only by kill action.